deploy/role.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  creationTimestamp: null
   name: project-initialize
 rules:
 - apiGroups:
```

Comment:
Considerations for `resourcequotas` must be made

Reply:
Added